unzip/test/test_o.c

```diff
@@ -25,7 +25,7 @@
  */
 #include "test.h"
 
-/* Test o arg - overrite existing files */
+/* Test o arg - override existing files */
```

Comment:
`overwrite`?

Reply:
Yes, makes more sense in this context. Thanks!